backend/flask_api.py

```diff
@@ -143,7 +142,12 @@ def get_projects():
     else:
         return make_response('Invalid parameters', 400)
 
-    res = make_response(dumps(res))
+    entries = res[:]
+    resArr = []
+    for entry in entries:
+        resArr.append(entry)
```

Comment:
For future cases, where a quick merge is desired and the code itself is solid (like this one), I strongly recommend adding a `TODO` comment to these lines really quick or fill an issue and link it here, it is okay to change other peoples branches for that reason. Tracking those smaller required changes to keep the code clean(er) is otherwise really difficult.